### PR TITLE
Add API docs for user endpoints

### PR DESCRIPTION
**Pull request by dannimajdalani**
Added users documentations

### CODE REVIEW

File: content/api/users/index.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+| `ShowEmail` | `boolean`                             | Allow others to see this email address? |
+| `Gender`    | `string`                              | `m`,`f`, or `u` for unspecified         |
+| `Title`     | `string`                              | Freeform user title                     |
+| `Location`  | `string`                              | Freeform user location                  |
+| `Role`      | [`multi smart id`](../multi smart-id) | Role ID (corresponds to GDN_Role)       |
+| `Rank`      | [`smart id`](../smart-id)             | Rank ID (corresponds to GDN_Rank)       |
+
+### Notes
+
+The Photo field submitted here is only able to take a remotely hosted URL, not a file upload. In order to upload an avatar to the service, use the [/users/photo API](http://docs.vanillaforums.com/api/users/)
```
Comment (slafleche) [inline content/api/users/index.md]: Your links are broken here. Don't use the production domain name. Hugo has a way to reference internal pages:

https://gohugo.io/content-management/cross-references/

**Comment by dannimajdalani** [comment]
I used typora. I will update with the above

**Comment by slafleche** [comment]
You can use Typora for writing our the code, but you need to test it in the site.

File: content/api/ignore/index.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+## /ignore/list
+
+```http
+GET /api/v1/ignore/list.ext HTTP/1.1
+Host: https://yoursite.vanillaforums.com
+```
+
+Get a user’s ignore list.
+
+[__Authentication__](../#making-api-calls): required
```
Comment (slafleche) [inline content/api/ignore/index.md]: Don't use "../" here. Remember, I told you to use https://gohugo.io/content-management/cross-references/
If we ever move this page under another page, all of these links will break.
`{{< relref "/api/index.md#making-api-calls" >}}`

If this link shows up again and again, let's make a shortcode for it. I'll help you.

Comment (slafleche) [reply content/api/ignore/index.md]: Forget about the shortcode, it seems like the relref doesn't play nice with shortcodes.

**Pull request merged**